stereoDNN/sample_app/networks.h

```diff
@@ -0,0 +1,36 @@
+
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+// Full license terms provided in LICENSE.md file.
+
+#pragma once
```

Comment:
We use header guards, not pragma once in the rest of the code so let's stick to that.

Reply:
done